configure.ac

```diff
@@ -2718,7 +2718,7 @@ AC_REPLACE_FUNCS(\
 	tempnam \
 )
 
-AC_CHECK_DECLS([getaddrinfo,getnameinfo,inet_ntop,inet_pton,InetNtopA,InetPtonA],,,[
+AC_CHECK_DECLS([InetNtopA,InetPtonA],,,[
```

Comment:
I have been reviewing the Sockets API fro MinGW recently and it seems modern Windows API (Vista+) is included in the set which provides the POSIX syscalls. I think we can drop these ```InetPtonA``` and ```InetNtopA``` as well.

Reply:
removed